assets/js/pages/ChecksCatalog/ChecksCatalog.jsx

```diff
@@ -95,9 +111,12 @@ function ChecksCatalog({
     },
     {
       optionsName: 'cluster-types',
-      options: clusterTypes.map((clusterType) => ({
-        value: clusterType,
-        disabled: !hasChecksForClusterType(completeCatalog, clusterType),
+      options: clusterCatalogFilters.map(({ type, hanaScenario }) => ({
+        value: { type, hanaScenario },
+        key: `${type}_${hanaScenario}`,
+        disabled:
+          !hasChecksForClusterType(completeCatalog, type) ||
```

Comment:
This double negation and the "or" almost burned my brain thinking the possibles outcomes XD
But I looks correct hehe
Maybe at some point (in next PRs) we can improve how to get this `disabled` value, because it is not simple at all

Reply:
Yeah, had also to play with this one a lot, in the end i double checked it by providing the correct checks for manual testing but thanks to the failing test, which pointed here :)